deploy/kubernetes/hcloud-csi-master.yml

```diff
@@ -132,11 +139,11 @@ spec:
               add: ["SYS_ADMIN"]
             allowPrivilegeEscalation: true
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v1.6.0
+          image: k8s.gcr.io/sig-storage/csi-provisioner:v2.2.2
           args:
-            - --provisioner=csi.hetzner.cloud
```

Comment:
[Deprecated option removed in external-provisioner 2.0](https://github.com/kubernetes-csi/external-provisioner/blob/master/CHANGELOG/CHANGELOG-2.0.md#no-really-you-must-read-this-before-you-upgrade).